src/lib/configs/project/fluence.ts

```diff
@@ -62,12 +63,19 @@ const configSchemaV0: JSONSchemaType<ConfigV0> = {
   required: ["version", "services"],
 };
 
+export const DISTRIBUTION_EVEN = "even";
+export const DISTRIBUTION_RANDOM = "random";
+export const DISTRIBUTIONS = [DISTRIBUTION_EVEN, DISTRIBUTION_RANDOM] as const;
```

Comment:
Why not use `enum` for this?

Reply:
There are multiple reasons why unions are superior to enums. Basically, the only real feature of enums (some people think it's one of their main flaws) is that they are opaque (means that if you have two of exactly the same enum structurally but defined in different modules - they will not be the same)

Please take a look at these two articles which completely persuaded me to stop using enums:
https://stackoverflow.com/questions/40275832/typescript-has-unions-so-are-enums-redundant
https://www.bam.tech/article/should-you-use-enums-or-union-types-in-typescript

Reply:
Sounds reasonable. Although I would put more points for rename refactoring part :)

Reply:
As far as I can see now, things changed and unions are pretty refactorable as well